findmy/reports/anisette.py

```diff
@@ -169,6 +170,7 @@ def __init__(self, server_url: str) -> None:
         self._http = HttpSession()
 
         self._anisette_data: dict[str, str] | None = None
+        self._anisette_data_stored_at: float | None = None
```

Comment:
I think this should be initialized to `0` or `-1` instead of None; either that or check for None in the if statement. I cannot see the precommit output from my phone, but I don't think pyright will be happy with how it is right now ;-)